wasi/testdata/args.wat

```diff
@@ -9,7 +9,7 @@
 (module
   (import "wasi_snapshot_preview1" "args_sizes_get" (func $wasi_args_sizes_get (param i32 i32) (result i32)))
   (import "wasi_snapshot_preview1" "args_get"       (func $wasi_args_get (param i32 i32) (result i32)))
-  (memory 0x200)  ;; just an arbitrary size big enough for tests
+  (memory 512)  ;; just an arbitrary size big enough for tests
```

Comment:
this is the page size so maybe this is too big.. 😄  maybe single digits or at most tens is enough I think 😄

Reply:
oh interesting. I was wondering and thought maybe it was bytes!